package.json

```diff
@@ -46,5 +44,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "react-flags": "^0.1.18"
```

Comment:
Why is this in dev dep?

Reply:
I really don't know she found herself there.